dv-export/python/dv_export.py

```diff
@@ -0,0 +1,191 @@
+            raise Exception(
+                f"Failed to get export metadata for export_id={export_id} response={repr(metadata_response)}")
+        logger.info(f"Successfully retrieved metadata for export_id={export_id}")
+        return metadata_response.json()
+
+    def _download_table_files_to_dir(self,
+                                     export_uuid: str,
+                                     tbl_name: str,
+                                     file_infos: list[FileInfo],
+                                     download_directory: str) -> list[str]:
+        file_paths = []
+        for file_info in file_infos:
+            logger.info(f"Downloading file={file_info.name} with id={file_info.file_id} for table={tbl_name}")
+            with self.client.get_export_file(export_uuid, file_id=file_info.file_id) as r:
+                r.raise_for_status()
+                file_path = download_directory + file_info.name
+                with open(file_path, 'wb') as f:
+                    for chunk in r.iter_content(chunk_size=None):
+                        if chunk:
+                            f.write(chunk)
+                file_paths.append(file_path)
+            logger.info(f"Downloaded file={file_info.name} with id={file_info.file_id} "
+                        f"for table={tbl_name} to file={file_path}")
+        return file_paths
+
+    def _create_and_populate_new_table(self,
+                                       table: TableInfo,
+                                       column_and_file_info: ColumnInfoAndFileInfo,
+                                       download_directory: str,
+                                       export_id: str):
+        self.data_store.create_initial_table_definition(table)
+        self._download_files_and_insert_into_table(table.name,
+                                                   column_and_file_info,
+                                                   download_directory,
+                                                   export_id)
+
+    def _download_files_and_insert_into_table(self,
+                                              table_name: str,
+                                              column_and_file_info: ColumnInfoAndFileInfo,
+                                              download_directory: str,
+                                              export_id: str):
+        local_files_for_table = self._download_table_files_to_dir(export_id,
+                                                                  table_name,
+                                                                  column_and_file_info.file_infos,
+                                                                  download_directory)
+
+        self.data_store.insert_from_files_into_table(table_name,
+                                                     column_and_file_info.column_infos,
+                                                     local_files_for_table)
+
+    def process_metadata(self,
```

Comment:
Docstring and Nit: consider a minor refactoring to make this more compact and readable